### ISSUE TITLE
Livestreamer refuses to grab source from recent Twitch TI broadcast

### ISSUE THREAD

**Issue by ptkovsky** [chrippa/livestreamer#1428]
`livestreamer -p /Applications/mpv.app/Contents/MacOS/mpv https://www.twitch.tv/dota2ti best
[cli][info] Found matching plugin twitch for URL https://www.twitch.tv/dota2ti
error: The specified stream(s) 'best' could not be found.
       Available streams: 1080p60, 360p30, 720p60, 144p30, 480p30, 720p30, 540p30, 240p30, audio`
but
`livestreamer -p /Applications/mpv.app/Contents/MacOS/mpv https://www.twitch.tv/dota2ti 1080p60
[cli][info] Found matching plugin twitch for URL https://www.twitch.tv/dota2ti
[cli][info] Available streams: 1080p60, 360p30, 720p60, 144p30, 480p30, 720p30, 540p30, 240p30, audio
[cli][info] Opening stream: 1080p60 (hls)
[cli][info] Starting player: /Applications/mpv.app/Contents/MacOS/mpv`

**Comment by ThisIsIvan** [comment]
`livestreamer twitch.tv/dota2ti 1080p60`  
should work

**Comment by ptkovsky** [comment]
Yeah, I posted it myself, are you reading?

Sure, that's what most people will do to watch it for now. But this problem makes things like UIs break too, and that's kinda annoying, especially for livestreamer users who are used to things like [livestreamer-twitch-gui](https://github.com/bastimeyer/livestreamer-twitch-gui) or are not sure how to use console commands.

Nvm, I think it got fixed already https://github.com/bastimeyer/livestreamer-twitch-gui/issues/283#issuecomment-238303191.
